sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -19,6 +19,7 @@
 import java.nio.ByteOrder;
 
 import org.apache.spark.memory.MemoryMode;
+import org.apache.spark.sql.execution.vectorized.ColumnVector.Array;
```

Comment:
@marmbrus , explicitly import the class here is to fix the scaladoc compile error. For the details you could see the above description.